manual/manual/library/builtin.etex

```diff
@@ -103,6 +103,7 @@ type int32
 \index{int32@\verb`int32`}
 \begin{ocamldocdescription}
  The type of signed 32-bit integers.
+ Literals for 32-bit integers are suffixed by l.
```

Comment:
This needs not be part of this PR, but it would be user-friendly to give examples of literal values at each of those described types: `()`, `Exit`, `[1; 2; 3]`, `Some true`, `1l, -1l`, etc.

Reply:
It sounds like a good idea, but for a separate PR.